editoast/src/core/stdcm.rs

```diff
@@ -90,6 +93,15 @@ pub struct LightWorkSchedule {
     pub track_ranges: Vec<UndirectedTrackRange>,
 }
 
+/// Lighter description of a work schedule with only the informations relevant for core
+#[derive(Debug, Serialize)]
+pub struct LightTemporarySpeedLimit {
```

Comment:
Not sure of the name here. To be honest, we already use `Light` in `editoast` to return something with less information because the original object is too huge (see `RollingStock` which contains `EffortCurve`). But I was trying to understand at first why we would need to do that in API's `editoast`. In this case, it's not really a light version, it's just the exact object for `core` (which happens to be a subset of the one exposed in `editoast` API). So I would just name `TemporarySpeedLimit` since we are in `core::stdcm` module and that we are describing the API to `core`, not something relative to `editoast` API. What do you think?

Reply:
Thank you for your comment. The `LightTemporarySpeedLimit` is indeed the exact object that core needs but it is also a lighter version of the `TemporarySpeedLimit` struct in `models`: whether it is a light version or not depends if we take `core` or `editoast` point of view. If we rename it to `TemporarySpeedLimit` I think it would lead to confusion and would require to make aliases for one of the two structs named this way that are used in `stdcm.rs`: we might aswell keep them named differently.  Maybe we could rename it to something like `CoreTemporarySpeedLimit` ? That name would be more explicit than `LightTemporarySpeedLimit` about the fact that this is the full `core` representation of a temporary speed limit. If we rename it we should also rename in the same manner the `LightWorkSchedule` just above it in stdcm.rs. What do you think ?

Reply:
Didn't notice for `LightWorkSchedule` so yes, indeed, if you'd be kind enough to change both, that'd be nice. For the naming, `CoreTemporarySpeedLimit` or `TemporarySpeedLimit` both work even if I believe that the latter is better. Aliasing is a problem of the consumer, but I don't think that should drive the design. Also, `clippy` tends to reject naming things like `core::CoreTemporarySpeedLimit` (see [`module_name_repetitions`](https://rust-lang.github.io/rust-clippy/master/index.html#/module_name_repetitions) but it is a `pedantic` category).